test/core.spec.js

```diff
@@ -1475,6 +1475,45 @@ describe('cli', function () {
       expect(gitArgs).toHaveLength(0);
     });
 
+
+    it('--signedoff adds signed-off-by to the commit message', async function () {
```

Comment:
It’s late here so maybe I’m missing something, but I can’t see how this test confirms what it says in the it statement- is it supposed to?

Reply:
I copied the previous test that was testing for the `--sign` option and how it affected the `git commit` command; 

Just to confirm - admit I've not dug into deep details of how the tests are structured - I've done the 'inverse' test and deliberately broke the code and the test flags this up...

```
  ● cli › with mocked git › --signedoff adds signed-off-by to the commit message

    expect(received).toEqual(expected) // deep equality

    - Expected  - 1
    + Received  + 1

    @@ -1,8 +1,8 @@
      Array [
        "commit",
    -   "--signoff",
    +   "--wibble",
        "CHANGELOG.md",
        "package.json",
        "package-lock.json",
        "-m",
        "chore(release): 1.0.1",
  ```

Gives me some confidence that test would act as a regression check.

Reply:
Aha, yep, makes sense. I’m not the original author, so I’m not across exactly how all the tests work